test/topojson/test.ml

```diff
@@ -61,12 +61,10 @@ let pp_position ppf t =
   let lng = Position.lng t in
   Fmt.pf ppf "[%f, %f]" lat lng
 
-let foreign_member ppf v =
-  let open Topojson.Geometry.Point in
-  let foreign = "arcs" in
-  Fmt.pf ppf "[%s]" foreign
+  let pp_foreign_member ppf (v : (string * Ezjsonm.value) list) =
+    Fmt.pf ppf "%a" Fmt.(list (pair string pp_ezjsonm)) v
```

Comment:
You will need to provide an implementation for `pp_ezjsonm` I left it out of my review :))